cmd/pipeline/update_pipeline.go

```diff
@@ -120,14 +149,42 @@ func NewCmdUpdatePipeline(config *cfg.Config) *cobra.Command {
 				format = cloud.ConfigFormatINI
 			}
 
+			sut := cloud.HPAScalingPolicyType(scaleUpType)
+			sdt := cloud.HPAScalingPolicyType(scaleDownType)
+
 			update := cloud.UpdatePipeline{
 				AutoCreatePortsFromConfig: &autoCreatePortsFromConfig,
 				SkipConfigValidation:      skipConfigValidation,
 				ConfigFormat:              &format,
 				Secrets:                   secrets,
 				Files:                     updatePipelineFiles,
-				Metadata:                  metadata,
+				Metadata:                  metadata,			
+			}
+
+			if minReplicas > 0 {
```

Comment:
This won't allow to update it back to zero.
Use `flags.Changed()` like here:

https://github.com/chronosphereio/calyptia-cli/blob/54397997cd84e13dcc06c5caadc430bc9385e01c/cmd/fleet/get_fleet.go#L35-L47

Reply:
Same for the rest of the fields.